server.js

```diff
@@ -1,5 +1,19 @@
-const app = require('./app')
+const mongoose = require("mongoose");
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
-})
+const app = require("./app");
+
+const DB_HOST =
+  "mongodb+srv://helenh:5bFOAtoZCnobJhHb@cluster0.5vw4gzu.mongodb.net/contacts_base";
```

Comment:
this is sensitive info. add it to .env file

Reply:
I fixed.